runtime/compiler/build/toolcfg/gnu/common.mk

```diff
@@ -125,7 +125,13 @@ CX_OPTFLAG?=$(CX_DEFAULTOPT)
 CX_FLAGS_PROD+=$(CX_OPTFLAG)
 
 ifeq ($(HOST_ARCH),x)
-    CX_FLAGS+=-mfpmath=sse -msse -msse2 -fno-math-errno -fno-rounding-math -fno-trapping-math -fno-signaling-nans
+    CX_FLAGS+=-mfpmath=sse -msse -msse2 -fno-math-errno -fno-trapping-math
+
+    # -fno-rounding-math and -fno-signalling-nans 
+    # optimization options are not supported on OSX
+    ifneq ($(OS),osx)
+        CX_FLAGS+=-fno-rounding-math -fno-signaling-nans
```

Comment:
This can be removed for GCC, as both are default, per https://gcc.gnu.org/onlinedocs/gcc/Optimize-Options.html